src/models/subscription.ts

```diff
@@ -79,21 +79,37 @@ export class Subscription extends Model {
 	}
 
 	static getAllForInstallation(
-		installationId: number
+		installationId: number,
+		gitHubAppId?: number
 	): Promise<Subscription[]> {
 		return this.findAll({
 			where: {
-				gitHubInstallationId: installationId
+				gitHubInstallationId: installationId,
+				...(gitHubAppId ? {
+					gitHubAppId
+				} : {
+					gitHubAppId: {
+						[Op.is]: null
+					}
```

Comment:
can't this just be represented with 
```
this.findAll({
			where: {
				gitHubInstallationId: installationId,
        gitHubAppId: gitHubAppId || null
	}
});
```
?

Reply:
I have no idea what I was doing, lol.
fixed.